### ISSUE TITLE
WCAG issues

### ISSUE THREAD

**Issue by pinggi** [mozilla/pdf.js#15635]
Firefox shows WCAG issues in Accessibility tab for pdf.js generated html:

1. added `canvas` element per pdf page 

```
Does not meet WCAG standards for text alternative.
```
link https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#content_with_images_must_be_labeled

2. `section` element with `textLayer` class
```
Does not meet WCAG standards for keyboard accessibility.
```
link https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics
